include/Utils/DIPUtils.h

```diff
@@ -563,12 +842,13 @@ void traverseImagewBoundaryExtrapolation(
   // Create constant indices.
   Value c0 = rewriter.create<ConstantIndexOp>(loc, 0);
   Value c1 = rewriter.create<ConstantIndexOp>(loc, 1);
-
   IntegerType i1 = IntegerType::get(ctx, 1);
 
   // Create DimOp.
   Value inputRow = rewriter.create<memref::DimOp>(loc, input, c0);
   Value inputCol = rewriter.create<memref::DimOp>(loc, input, c1);
+  Value outputrow = rewriter.create<memref::DimOp>(loc, output, c0);
```

Comment:
What is the use of these variables? Please remove them if they are not of any use.